force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -133,10 +177,20 @@ public with sharing class VariablesDataTypesOperators {
      * @param dt The date to check.
      * @return True if the date is in the past, False otherwise.
      */
-    public static Boolean isDateInPast(Date dt) {
-        return null; // Replace null with the variable you used to store the result
-    }
-
+     public static Boolean isDateInPast(Date dt) {
+         //dt = Date.newInstance(2016, 6, 24);
+         Date today = Date.today();
+         if (dt < today)
+         {
+             return true;
+         }
+         else 
+         {
+             return false;
+         }
```

Comment:
Again, think about how we can combine all of these lines into a single line of code.

Reply:
I'm not sure about this one. Would it be 
return (dt < today);

Reply:
Yes, `return dt < today` is better -- but we could condense it further to:
`return dt < Date.today();`